.storybook/stories/useFBXLoader.stories.js

```diff
@@ -0,0 +1,41 @@
+import React, { Suspense } from 'react'
+
+import { Setup } from '../Setup'
+
+import {useFBXLoader} from "../../src/loaders/useFBXLoader";
+import {useCubeTextureLoader} from "../../src";
+import {Vector2} from "three";
+
+export default {
+  title: 'Loaders/FBX',
+  component: useFBXLoader,
+  decorators: [(storyFn) => <Setup cameraPosition={[0, 0, 5]}>{storyFn()}</Setup>],
+}
+
+function SuzanneFBX() {
+  let  fbx = useFBXLoader('suzanne/suzanne.fbx')
+
+  const envMap = useCubeTextureLoader(['px.png', 'nx.png', 'py.png', 'ny.png', 'pz.png', 'nz.png'], { path: 'cube/' })
```

Comment:
I would remove this envMap to keep the story simple

Reply:
sure will do